scripts/analysis.py

```diff
@@ -663,7 +804,51 @@ def plot_power(cursor):
     con = lite.connect(file)
     with con:
         cur = con.cursor()
-        print(snf(cur))
-        plot_power(cur)
-        plot_in_out_flux(cur, 'source', False, 'source vs time', 'source')
-        plot_in_out_flux(cur, 'sink', True, 'isotope vs time', 'sink')
+        # print(snf(cur))
+        # plot_power(cur)
+        # plot_in_out_flux(cur, 'source', False, 'source vs time', 'source')
+        # plot_in_out_flux(cur, 'sink', True, 'isotope vs time', 'sink')
+        init_year, init_month, duration, timestep = get_sim_time_duration(cur)
+        """
```

Comment:
Avoid commenting out code in general (this is under version control. if you commit often, you won't lost it). but if you must, then please use # rather than docstring here. (reason: autodocumentation headaches)